Kickstarter-iOS/ViewModels/RootViewModel.swift

```diff
@@ -289,3 +289,10 @@ private func first<VC: UIViewController>(_ viewController: VC.Type) -> ([UIViewC
       .flatMap { viewControllers[$0] as? VC }
   }
 }
+
+private func profileController() -> UIViewController {
+  let showDash = AppEnvironment.current.config?.features["ios_backer_dashboard"] == .some(true)
```

Comment:
could we add a test for this? we have a few examples of testing features, but basically you just push a new config onto the environment:

```swift

    let config = .template
      |> Config.lens.features .~ ["ios_backer_dashboard": true]

    withEnvironment(config: config) {
```